tests/integration/components/mutually_exclusive/test_checkbox_single_checkbox_override.py

```diff
@@ -0,0 +1,45 @@
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestCheckboxSingleCheckboxOverride(IntegrationTestCase):
+    """
+    Tests to ensure that the server-side validation for mutually exclusive checkbox
+    with single checkbox override function as expected. These tests emulate the non-JS version.
+    """
+
+    def setUp(self):
+        super().setUp()
+        self.launchSurvey('test', 'mutually_exclusive_single_checkbox_override')
+
+    def test_non_exclusive_answer(self):
+        # When
+        self.post({'checkbox-answer': ['British', 'Irish']})
+
+        # Then
+        self.assertInUrl('section-summary')
+        self.assertInPage('British')
```

Comment:
For this one (and all others in this PR), it should be assertInBody.  We were going to make this change as part of https://trello.com/c/7xsxQmNz/2407-change-assertinpage-to-assertinbody and doing it now would reduce the amount of work we'd have to do

Reply:
Cool, will change.

Reply:
Just checked, I thought assertInPage/assertInBody were standard functions in unittest.
Yea, ignore that comment as I thought it was simple to do.